### PR TITLE
Integrated Hotjar and Cookiebot tag and Clicked Data attribute variable from the MTM extended plugin, #PG-3596

### PR DESCRIPTION
**Pull request by AltamashShaikh**
### Description:

Integrated Hotjar and Cookiebot tag and Clicked Data attribute variable from the MTM extended plugin
Fixes: #PG-3596

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: Template/Tag/HotjarTag.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                $field->description = Piwik::translate('TagManager_HotjarHJIDDescription');
+                $field->customFieldComponent = self::FIELD_VARIABLE_COMPONENT;
+                $field->validators[] = new NotEmpty();
+            }),
+
+            $this->makeSetting('hjsv', 6, FieldConfig::TYPE_INT, function (FieldConfig $field) {
+                $field->title = Piwik::translate('TagManager_HotjarHJSV');
+                $field->description = Piwik::translate('TagManager_HotjarHJSVDescription');
+                $field->customFieldComponent = self::FIELD_VARIABLE_COMPONENT;
+                $field->validators[] = new NotEmpty();
```
Comment (snake14) [inline Template/Tag/HotjarTag.php:72]: Should this be numeric?

**Review by snake14 — APPROVED** [review]
Looks good. I confirmed that both tags loaded as expected and the data attribute variable worked as expected.

File: Template/Tag/HotjarTag.php (hunk 1 above)
Comment (AltamashShaikh) [reply Template/Tag/HotjarTag.php:72]: @snake14 Ideally this should be fixed to 6, but they have made it configurable and I am not sure if this should be only numeric, but good to make it numeric :+1:

Comment (AltamashShaikh) [reply Template/Tag/HotjarTag.php:72]: Done :+1:

**Review by snake14 — APPROVED** [review]
Changes look good and confirmed that it requires the value to be numeric :+1:

**Pull request merged**